src/browser/ui/__tests__/ReactMount-test.js

```diff
@@ -144,4 +144,13 @@ describe('ReactMount', function() {
     ReactMount.render(<div />, container);
     expect(console.warn.mock.calls.length).toBe(0);
   });
+
+  it('should warn when mounting into document.body', function () {
+    spyOn(console, 'warn')
+    ReactMount.render(<div />, document.body);
```

Comment:
Yeah, the browser mode is a little bit of a hidden feature. We haven't been great at keeping it up, but we may as well try to avoid breaking it more. :)